src/config_legacy.cpp

```diff
@@ -245,6 +245,7 @@ namespace ConfigLegacy
         uint8_t analogAdcPinY;
         uint8_t forced_circularity;
         uint8_t analog_deadzone;
+        uint8_t auto_calibrate;
```

Comment:
You should not change `config_legacy.cpp` at all. These `struct`s represent the final state of our config storage before the migration to Protobuf. Changing their structure will break the migration from pre-Protobuf config to Protobuf config.